tran.c

```diff
@@ -119,7 +119,7 @@ void arginit(int ac, char **av)	/* set up ARGV and ARGC */
 	for (i = 0; i < ac; i++) {
 		double result;
 
-		sprintf(temp, "%d", i);
+		snprintf(temp, "%d", i);
```

Comment:
You are missing a sizeof(temp) as the 2nd parameter to snprintf().  Please test-compile your PRs.